src/types/annotator.js

```diff
@@ -61,7 +61,7 @@
  *
  * @typedef AnchoringImpl
  * @prop {(root: HTMLElement, selectors: Selector[], options: any) => Promise<Range>} anchor
- * @prop {(root: HTMLElement, range: Range, options: any) => Promise<Selector[]>} describe
+ * @prop {(root: HTMLElement, range: Range, options: any) => Selector[]|Promise<Selector[]>} describe
```

Comment:
The HTML anchoring implementation has a synchronous `describe` method, whereas PDF anchoring has an async one. We might want to force both implementations to use the same interface in future. For now I've simply updated the types to match the implementation. Previously this was wrong but TypeScript wasn't able to see that.